applications/CompressiblePotentialFlowApplication/custom_processes/compute_embedded_lift_process.cpp

```diff
@@ -46,19 +46,15 @@ void ComputeEmbeddedLiftProcess<Dim, NumNodes>::Execute()
         auto it_elem=mrModelPart.ElementsBegin()+i;
         auto r_geometry = it_elem->GetGeometry();
 
-        BoundedVector<double,3> geometry_distances;
+        BoundedVector<double, NumNodes> elemental_distances;
         for(unsigned int i_node = 0; i_node<NumNodes; i_node++){
-            geometry_distances[i_node] = r_geometry[i_node].GetSolutionStepValue(GEOMETRY_DISTANCE);
+            elemental_distances[i_node] = r_geometry[i_node].GetSolutionStepValue(GEOMETRY_DISTANCE);
```

Comment:
This change in the variable name is misleading. I'd rather keep the old one since as `elemental_distances` what comes to my mind is a discontinuous level set, and you are retrieving a continuous one from the nodes.

Reply:
Yes, you are right!